spec/models/job_template_spec.rb

```diff
@@ -1,9 +1,25 @@
 require 'spec_helper'
 
 describe JobTemplate do
-  it { should have_many(:steps) }
-  it { should respond_to?(:cluster_job_templates) }
-  it { should respond_to?(:environment) }
+  it 'has many steps' do
+    expect(subject).to have_many(:steps)
+  end
+
+  it 'responds to .cluster_job_templates' do
+    expect(described_class.respond_to?(:cluster_job_templates)).to be_truthy
+  end
+
+  it 'responds to #environment' do
+    expect(subject.respond_to?(:environment)).to be_truthy
+  end
+
+  it 'responds to #vendor' do
+    expect(subject.respond_to?(:vendor)).to be_truthy
+  end
+
+  it 'responds to #adapter' do
+    expect(subject.respond_to?(:adapter)).to be_truthy
+  end
```

Comment:
When I added vendor and adapter specs following the style of the existing code
```
it { should respond_to?(:vendor) }
it { should respond_to?(:adapter) }
```
the tests passed even before I made any code changes or migrations.  I lost faith in that style and switched them to the "normal" style shown here.  These last two failed until I generated and ran the migrations as expected.